global_recon/models/global_recon_model.py

```diff
@@ -99,7 +100,14 @@ def init_data(self, in_dict):
                 new_dict['invis_frames'] = invis_frames = visible == 0
                 new_dict['frame2ind'] = {f: i for i, f in enumerate(new_dict['frames'])}
                 new_dict['scale'] = None
-                smpl_pose_wroot = quaternion_to_angle_axis(torch.tensor(pose_dict[f'smpl_pose_quat_wroot'], device=self.device)).cpu().numpy()
```

Comment:
Thank you for you PR and it does save me a lot of time!
However, I think modification here may not be suitable.
In the [issue](https://github.com/NVlabs/GLAMR/issues/34#issuecomment-1671664667) you mentioned, the answerer said that "This probably happens because you use the hrnet version of HybrIK which produces 3x3 rotation matrices, not quaternions". And in my case, I do receive quaternions as the output, which means modification here actually make my code fail.